examples/inverse_kinematics_example/setup.cfg

```
@@ -0,0 +1,4 @@
+[develop]
+script_dir=$base/lib/inverse_kinematics
+[install]
+install_scripts=$base/lib/inverse_kinematics
```

Seems like this has to change to allow `ros2 run` to find the executable for the node.

```suggestion
script_dir=$base/lib/inverse_kinematics_example
[install]
install_scripts=$base/lib/inverse_kinematics_example
```